Makefile

```diff
@@ -55,6 +55,7 @@ test_images:
 	sudo -E ./tests/test_cli.sh tests/cli/test_compose_ext4-filesystem.sh  \
 				    tests/cli/test_compose_partitioned-disk.sh \
 				    tests/cli/test_compose_tar.sh              \
+				    tests/cli/test_compose_tar_kickstart.sh    \
```

Comment:
FYI not really needed. In fact this target is being removed in #885.

Reply:
OK, I'll get rid of it.